ci/tests/puppeteer/run.sh

```diff
@@ -51,8 +51,10 @@ random=$(openssl rand -hex 8)
 
 if [[ ! -d "$PWD"/ci/tests/puppeteer/node_modules/puppeteer ]] ; then
   printgreen "Installing Puppeteer"
-  npm_install_cmd="npm i --prefix "$PWD"/ci/tests/puppeteer"
```

Comment:
For the npm install, on windows with msys2 bash, if I ran ```npm_install_cmd="npm i --prefix "$PWD"/ci/tests/puppeteer" "$PWD"/ci/tests/puppeteer"``` (adding directory argument in addition to prefix) then it worked but it put all the npm scripts directly in the prefix folder. If I ran it without --prefix or without the directory argument it didn't work. The fix for me was to cd into the folder and run npm install there.